demosplan/DemosPlanCoreBundle/Logic/Procedure/ProcedureService.php

```diff
@@ -1123,10 +1123,6 @@ public function updateProcedureObject(Procedure $procedureToUpdate)
             $sourceProcedure->setSettings($sourceProcedureSettings);
 
             $procedure = $this->procedureRepository->updateObject($procedureToUpdate);
```

Comment:
I don't understand this approach. `updateObject` simply validates and persists the changes that must already be present in `$procedureToUpdate`. So cloning it before will result in two instances without differences after the update.

Reply:
Currently the method is only called in the context of `ProcedureService::resetMapHint()`. There the `mapHint` is set under the procedure settings. I tested this case locally and debuged into the event, and indeed no changes between the procedures. I replaced the use of the method `ProcedureService::updateProcedureObject()` in [eb48b27](https://github.com/demos-europe/demosplan-core/pull/1741/commits/eb48b2765f099161068d696dc3a6fab0c1e16747) with `ProcedureService->updateProcedure()`. I added a `todo` to remind that the `ProcedureService::updateProcedureObject()` has to be refactored. What do you think.